### PR TITLE
Migrating terminal module to the new ev-engine

### LINKED ISSUE #11: Make the hardcoded selector configurable
https://github.com/jillix/engine-terminal/blob/5d86b7e677abe278b6543df35e60136041ca2e23/client/index.js#L12

### LINKED ISSUE #12: Module specific CSS
- `whitespace`
- height, width etc

### PR DESCRIPTION
**Pull request by lcristianiim**
Migrated the terminal module to the new Engine version.

Fixes #11, fixes #12.

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ var $ = require("/jquery");
  * @return {undefined}
  */
 exports.init = function () {
-    this._term = $("#terminal").webTerm(this);
+    // Use config.container
```
Comment (IonicaBizau) [inline client/index.js]: Remove this comment.

File: package.json [hunk 2]
```diff
@@ -38,7 +21,37 @@
...
+              "/jquery.js"
+          ],
+          "flow": [
+              {
+                  "on":"termData",
+                  "call": "terminal/termData"
+              }
+          ],
+          "styles": ["public/term.css"],
+          "markup": []
```
Comment (IonicaBizau) [inline package.json]: You can remove this `markup` field, since it's empty.

File: public/term.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
+div {
+    border: 0;
+    padding: 0;
+    margin: 0 0 0 0;
+}
+div.terminal > div {
+    padding-bottom: 1.35px;
+    white-space: nowrap;
+}
+#terminal {
```
Comment (IonicaBizau) [inline public/term.css]: I guess this can be removed too. It's application specific.

File: public/term.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    font-size: 11px;
+    line-height: 1;
+    color: #f0f0f0;
+    background: #18222C !important;
+}
+.reverse-video {
+    color: #000;
+    background: #f0f0f0;
+}
+#terminal .bar {
```
Comment (IonicaBizau) [inline public/term.css:10]: This, too. Check if that's fine.

**Comment by lcristianiim** [comment]
@IonicaBizau please check it now.
I have also cleaned the term.css

File: client/index.js (hunk 1 above)
Comment (IonicaBizau) [inline client/index.js]: /cc @lcristianiim And this one too! :smile:

**Comment by lcristianiim** [comment]
Cleaning code of unnecessary comment :smile: 
Thanks @IonicaBizau

**Comment by IonicaBizau** [comment]
:tada: :dizzy:

**Pull request merged**